### ISSUE TITLE
UI doesn't appear on Chromebook

### ISSUE THREAD

**Issue by ehmorris** [ehmorris/High-Resolution-Downloader-For-Instagram#54]
From a webstore support issue:

> This works fine on macOS 10.14.5 (with Chrome 81), but does not seem to work on my Pixelbook ChromeOS 81. When I open an IG story and click on the image, no Open in Tab or Download buttons appear.

**Comment by paulschreiber** [comment]
Let me know what I can do to provide additional information. Happy to try a debug build.

**Comment by paulschreiber** [comment]
On ChromeOS, the buttons appear, as expected, on standard, non-story posts.

**Comment by ehmorris** [comment]
@paulschreiber Sorry it's taken me so long to get back to you. It's unlikely that I'll be able to reproduce this any time soon since I don't have easy access to a Chromebook. If you have the inclination to clone this repo, generate a build, and then record the console errors when reproducing this issue, I might be able to fix it without the Chromebook.

I'm not sure where to start since story images and post images are formatted almost identically:

Post:

```
<img
  alt="Photo by Edwin in New Museum. Image may contain: sky, tree and outdoor."
  class="FFVAD"
  decoding="auto"
  sizes="480px"
  srcset="
    https://scontent-lga3-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/p640x640/81520477_785604581949848_3888193941092623666_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=108&amp;_nc_ohc=y_TxUyPeKe4AX_z2ufn&amp;_nc_tp=25&amp;oh=461abfdf961f2fe91d2590144a5a26dc&amp;oe=5FAF89C3  640w,
    https://scontent-lga3-1.cdninstagram.com/v/t51.2885-15/sh0.08/e35/p750x750/81520477_785604581949848_3888193941092623666_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=108&amp;_nc_ohc=y_TxUyPeKe4AX_z2ufn&amp;_nc_tp=25&amp;oh=42980af16653872bbd79c0956141ab89&amp;oe=5FAECE43  750w,
    https://scontent-lga3-1.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/81520477_785604581949848_3888193941092623666_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=108&amp;_nc_ohc=y_TxUyPeKe4AX_z2ufn&amp;_nc_tp=19&amp;oh=348e78c2098eb1f994be3262af2ab4ce&amp;oe=5FAE9F28      1080w
  "
  src="https://scontent-lga3-1.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/81520477_785604581949848_3888193941092623666_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=108&amp;_nc_ohc=y_TxUyPeKe4AX_z2ufn&amp;_nc_tp=19&amp;oh=348e78c2098eb1f994be3262af2ab4ce&amp;oe=5FAE9F28"
  style="object-fit: cover;"
/>
```

Story:

```
<img
  class="y-yJ5  "
  decoding="sync"
  sizes="529.875px"
  srcset="
    https://scontent-lga3-1.cdninstagram.com/v/t51.12442-15/sh0.08/e35/p640x640/54732246_275529746697334_6351449441454203496_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=110&amp;_nc_ohc=5ZggKQkEQDUAX-VaB-O&amp;_nc_tp=25&amp;oh=f1d0f315c0c9af5b6fa72510d67cd643&amp;oe=5F8778DB  640w,
    https://scontent-lga3-1.cdninstagram.com/v/t51.12442-15/sh0.08/e35/p750x750/54732246_275529746697334_6351449441454203496_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=110&amp;_nc_ohc=5ZggKQkEQDUAX-VaB-O&amp;_nc_tp=25&amp;oh=3533540a5cdbb64035e0e2b531cdadf6&amp;oe=5F8725DB  750w,
    https://scontent-lga3-1.cdninstagram.com/v/t51.12442-15/e35/54732246_275529746697334_6351449441454203496_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=110&amp;_nc_ohc=5ZggKQkEQDUAX-VaB-O&amp;_nc_tp=18&amp;oh=28453c2f4d2e5725a83311afd70b8739&amp;oe=5F87273E                 1080w
  "
  src="https://scontent-lga3-1.cdninstagram.com/v/t51.12442-15/e35/54732246_275529746697334_6351449441454203496_n.jpg?_nc_ht=scontent-lga3-1.cdninstagram.com&amp;_nc_cat=110&amp;_nc_ohc=5ZggKQkEQDUAX-VaB-O&amp;_nc_tp=18&amp;oh=28453c2f4d2e5725a83311afd70b8739&amp;oe=5F87273E"
/>
```

Let me know if you have time to do this. Appreciate the report regardless!

**Comment by ehmorris** [comment]
@paulschreiber it's possible you were on the newer version of IG stories. I've created a fix for that here: https://github.com/ehmorris/High-Resolution-Downloader-For-Instagram/pull/61

I'm going to be putting this in the Chrome story shortly. Let me know if that fixes the problem.

**Comment by paulschreiber** [comment]
@ehmorris Let me know when it's been deployed.

**Comment by ehmorris** [comment]
@paulschreiber looks like it's out now. Just make sure you're using `2.0.26` on the extensions page (chrome://extensions)

**Comment by paulschreiber** [comment]
The problem is still present with 2.0.26.

**Comment by paulschreiber** [comment]
I reproduced the problem with a build version, but don't see console errors. If you'd like to debug, I'd be happy to screen share over Google Meet or similar.

**Comment by paulschreiber** [comment]
This is fixed in 2.0.28.

**Comment by ehmorris** [comment]
Strange. Must have been related to the max top value. Thanks for updating me!
